javascript/javascript1/week2/formal-fullname.js

```diff
@@ -0,0 +1,28 @@
+
+
+function getFullname (firstname, surname, useFormalName, gender) {
+    if (!firstname || !surname)  {
+        return `"Please fill up firstname and surname."`
+    }
+    else {
```

Comment:
This `else` here is not necessary because you `return` inside the `if`.

Though if you choose to keep it you should indent the content of the content of `else`. Right now its content is (wrongly) on the same level as the `else` itself.